qa/L0_model_config/autofill_noplatform/python/conflicting_scheduler_sequence/expected

```diff
@@ -0,0 +1 @@
+Configuration already has scheduler 'sequence_batching' and will not auto-complete scheduler to dynamic_batching.
```

Comment:
configuration specified scheduling_choice as 'sequence_batching', but auto-complete-config function for model 'conflicting_scheduler_sequence' tries to set scheduling_choice as 'dynamic_batching'.

Reply:
To double check my understanding, you are suggesting a more comprehensive error statement, correct?

Reply:
correct. And this is in accordance with our other error statements for failing cases.

Reply:
fixed